example/w-hot-reloading/hot_reloading.ml

```diff
@@ -0,0 +1,89 @@
+let hotreload_script
+    ?(retry_interval_ms = 500)
+    ?(max_retry_ms = 5000)
+    ?(route = "/_hotreload")
+    ()
+  =
+  Printf.sprintf
+    {js|
+var socketUrl = "ws://" + location.host + "%s"
+var s = new WebSocket(socketUrl);
+
+s.onopen = function(even) {
+  console.log("WebSocket connection open.");
```

Comment:
Not familiar with developer expectations for this, but do we need logging in such a script? Personally, I would try to observe the activity from the Network tab, which I assume most browsers offer.

Reply:
I'm not sure either.. Personally, I feel reassured to have the logs tell me that everything is going well, but that might only be because I wrote the script.

The log is not useful in its current state though. It could be something like "Live-reload WebSocket connection established, waiting for the server to restart to reload the page."?

Reply:
🤷 . if it's there, perhaps it should be more specific, that it's the hot-reload socket, vs any general socket connection :)

Reply:
I still favor removing it, as:

- This is an example, and it's best to keep it brief, to make it easier to focus on the actual machinery.
- The information is redundant with the network tab, and direct observation that reloading continues to work.